depthai_ros_driver/src/conversion.cpp

```diff
@@ -0,0 +1,92 @@
+#include "depthai_ros_driver/conversion.hpp"
+
+cv::Mat rr::chw2hwc(const cv::Mat& mat) {
+    cv::Mat new_mat = cv::Mat(mat.size(), mat.type());
+    const auto WH = mat.cols * mat.rows;  // WH = (W * H)
```

Comment:
This is simpler. `n * WH` should be optimized and will not be done inside the loop.

```suggestion
    uchar* pSrc = mat.data;
    uchar* pDst = new_mat.data;
    for (int i = 0; i < WH; ++i) {
        pDst[3 * i + 0] = pSrc[0 * WH + i];
        pDst[3 * i + 1] = pSrc[1 * WH + i];
        pDst[3 * i + 2] = pSrc[2 * WH + i];
    }
```

Reply:
Thanks Kota-san